### ISSUE TITLE
Open-Source The Project

### ISSUE THREAD

**Issue by kataras** [builtbybel/LoveWindowsAgain#2]
Hello @Belim,

How would be like to open-source the application's source so 

1. people can trust it without a doubt
![image](https://user-images.githubusercontent.com/22900943/172029578-fc3c2265-90b5-4769-9e2e-916d2c515b66.png)
2. gain community support and bug-reports

**Comment by Belim** [comment]
Hey kataras,
I'm currently refactoring the project to enhance maintainability. After the project is complete 💯  it will be definetley pushed to GitHub and open sourced 😃 MIT is already registered as the license...

**Comment by Mezque** [comment]
> Hello @Belim,
> 
> How would be like to open-source the application's source so
> 
> 1. people can trust it without a doubt
>    ![image](https://user-images.githubusercontent.com/22900943/172029578-fc3c2265-90b5-4769-9e2e-916d2c515b66.png)
> 2. gain community support and bug-reports

All of the other work I've used from belim is open source, and I trust myself that he will publish the src once he feels its ready, if you wanted to you can look at previous projects from him for more of a reason to trust this one as well! Crome flags anything that isn't a recognized file as potentially dangerous and is nothing to need to worry about.

**Comment by Marcinoo97** [comment]
its written in c# so having binary is like having source as c# can be decompiled to source easily.  I just wonder why the creator choose 22598.200(insider build) and not 22621 (stable build to be released in November)

**Comment by Mezque** [comment]
> its written in c# so having binary is like having source as c# apps can be easily decompiled to source. I just wonder why the creator choose 22598.200(insider build) and not 22621 (stable build to be released in November). I'm confused how can this app can have MIT licence while actively breaking it.


the source is now available on the repo [here](https://github.com/builtbybel/LoveWindowsAgain/tree/main/src/LWA/Features).

**Comment by Belim** [comment]
allow me 2-3 days then it is complete 😃

**Comment by julianxhokaxhiu** [comment]
> its written in c# so having binary is like having source as c# apps can be easily decompiled to source. I just wonder why the creator choose 22598.200(insider build) and not 22621 (stable build to be released in November). I'm confused how can this app can have MIT licence while actively breaking it.

I don't want to derail this issue but this is the same question I had while I was going across this project. Is this going to support 22621 as well?

**Comment by Belim** [comment]
> I don't want to derail this issue but this is the same question I had while I was going across this project. Is this going to support 22621 as well?

Yes, this [build]((https://github.com/builtbybel/LoveWindowsAgain#requirements)) has been included yesterday

**Comment by julianxhokaxhiu** [comment]
Thank you!

**Comment by MrTJay** [comment]
I would really like an "undo" or uninstall for this hacks/tweaks!
I can't fix the issue now where my Windows 11 PC doesn't sleep!
